isaquetdiniz/buyList
# Feat/jest

## What I did

- Create suite of tests

## How to test

1. Open the terminal and go to the folder of this project

1. In terminal:
   1. `git checkout develop`
   1. `git pull`
   1. `git checkout feat/jest`
   1. `git pull origin feat/jest`
   1. `yarn install`
   1. Run the test database docker put your name to buytestlist
   1. Configure the .env
   1. Run the `yarn test`

1.Check if the tests are fine

1. Read the changed files section on GitHub 

1. Test the things that you think are worth testing, even the ones that are not in this description 

Resolves:
